docs/examples/pytorch/single_stage_detector/pytorch_ssd.rst

```diff
@@ -0,0 +1,76 @@
+Single Shot MultiBox Detector training in PyTorch
+============================
+
+This example implements Single Shot Multibox Detector originally published by Wei Liu, Dragomir Anguelov, Dumitru Erhan, Christian Szegedy, 
+Scott Reed, Cheng-Yang Fu, Alexander C. Berg as `SSD: Single Shot MultiBox Detector <https://arxiv.org/abs/1512.02325>`_.
+
+Code is based on MLPerf example from `here <https://github.com/mlperf/training/tree/master/single_stage_detector/ssd>`_ and has been modified to use DALI. 
+
+To run use following command:
+
+.. code-block:: bash
+
+   python train.py
+
+
+Requirements
+------------
+
+- This example was tested with :bash:`python3.5.2` and it should work with later versions. It will not work with :bash:`python2.7` and earlier.
```

Comment:
I have an impression, that this:
":bash:python3.5.2"
isn't displayed correctly. Could you verify?
Also it seems to be the case in some other places as well

Reply:
![ssd-doc](https://user-images.githubusercontent.com/44801854/49642648-f82fb980-fa13-11e8-9c8f-99265cd61803.png)

This is how it looks, I think it suffice.

Reply:
That's how it looks for me, maybe somebody else could verify?

![screenshot from 2018-12-07 11-38-07](https://user-images.githubusercontent.com/7101064/49642933-a0458280-fa14-11e8-9cdc-0f23d7c36d9b.png)

Reply:
Yeah, picture I pasted was from doc generated locally using sphinx, I guess github does not handle rst files well. if you see other .rst files (like in resnet50 example) there are also problems there. I think maybe effort to make all docs look good in github should be separate from this PR? If indeed we do want to do it.

Reply:
This works well https://github.com/NVIDIA/DALI/blob/master/docs/examples/pytorch/renet50/pytorch-resnet50.rst. I think you can use different tag.

Reply:
Try to use:
```
.. code-block:: bash
```
Seems that this inline code doesn't work in the GitHub.

Reply:
> This works well https://github.com/NVIDIA/DALI/blob/master/docs/examples/pytorch/renet50/pytorch-resnet50.rst. I think you can use different tag.

Under Training section there is :fileref: link visible, that is what I meant. 

I will apply different method to highlight python versions.

Reply:
Done, inline highlights look good both in sphinx generated version and in github using double grave accent method.